web-services/deploy/application/src/main/wildfly/add-datawave-configuration.cli

```diff
@@ -88,6 +88,7 @@ module add --name=com.mysql.driver --dependencies=javax.api,javax.transaction.ap
 /system-property=dw.metrics.reporter.port:add(value=${metrics.reporter.port})
 /system-property=dw.metrics.reporter.class:add(value=${metrics.reporter.class})
 /system-property=dw.cache.tableNames:add(value="${tables.to.cache}")
+/system-property=dw.cache.reloadInterval:add(value="${cache.reloadInterval}")
```

Comment:
I think we just need to add `cache.reloadInterval` to default.properties.
From the CI job log...
```
...
Operation: /system-property=dw.cache.reloadInterval:add(value="${cache.reloadInterval}")
Failure: WFLYCTL0211: Cannot resolve expression '${cache.reloadInterval}'

Failed to deploy Datawave configuration to Wildfly. Please see  for details.
...
````